packages/components/src/Text/Text.tsx

```diff
@@ -127,16 +101,15 @@ export type TypographyProps = {
    */
   color?: TypographyColor;
   /**
-   * The size of the html element. If no `as` prop is provided, then
-   * the component uses this value to determine which html tag to render
+   * The size of the html element. If used as a Heading and no `as` prop is provided,
+   * then the component uses this value to determine which html tag to render
    * (e.g. 'h1', 'h2', etc.)
    */
   size: TypographySize;
 };
 
-function Typography(props: TypographyProps): JSX.Element {
+function Text(props: TextProps): JSX.Element {
```

Comment:
I know this isn't part of the change here, but can/should we remove the `JSX.Element` return values from these components? I don't think they're giving us much, and TypeScript will happily infer the right thing (whether that's a string, an element, null, a number, etc).

Reply:
hm yeah I thought I added it b/c TS complained earlier but I'll check

Reply:
<img width="390" alt="Screen Shot 2021-02-05 at 3 14 00 PM" src="https://user-images.githubusercontent.com/15840841/107084244-ccabe380-67c4-11eb-98a5-0330a16f5b39.png">

ah it was ESLint that was unhappy. should we turn off this rule?

Reply:
Oh, I see. It wants explicit types on module boundaries (such as an export).

My opinion would be to disable rule. But it's fine either way, and probably not worth worrying about on this PR.

Reply:
okie i'll also do it in a follow-up